### ISSUE TITLE
the poisson noise fiasco

### ISSUE THREAD

**Issue by lmoffatt** [lmoffatt/macro_dr#121]
together with pink noise I added a supposedly proportional noise to the signal. The only thing is that I forgot that it was the variance we are talking about so the noise variance was proportional to the absolute value of the signal, imitating a poissonian process. 

to my shock this "proportional" parameter that was set to be 0.01 was consistently about 1-2. That is close to the value of the unitary current. 

I was very worried because this fact completly obliterate the purpose of macro_dr, model the gating noise. This single parameter effectively compete with macro_dr in all but one situation: the experiment_7 (only the last trace) for simulations of schemes 1 and 2. The proportional parameter (that was simulated in the simulations) was wrongly recovered in the 1-2 range even for simulations.

**Comment by lmoffatt** [comment]
### The light

I found that for cuevi experiments on simulations of scheme 1 of a single trace, the proportional parameter was 1 for low number of samples and 1e-2 for all the samples. Therefore: is it possible that it hijacks the fitting when there is not enough information for macro_dr to act?

How can we test this hipothesis?

**Comment by lmoffatt** [comment]
So, the first step is to try the following two new experiments

experiment_no_10 --> remove the 10 ms pulses. 

experiment_same_pulse -> make all the pulses big.

experiment_long  --> no semilog time scaling, get the full dosis of fluctuations.

**Comment by lmoffatt** [comment]
I thought long about this thing, and then I realized that the "Proportional_noise" is NOT a probability, it does not integrate to one. Even worst, it is difficult to find a probability hidden there. 

Lets try to normalize this probability. 
we have fixed the ymean and yvar and then the logL is a normal distribution with ymean and yvar +y*Poi

so lets itegrate for y from -inf to + inf

```math
\int {1 \over {\sqrt {2 \cdot \pi \cdot (\sigma_{var} + y \cdot Poi )}}}  \cdot \exp {- (y-y_{mean})^2 \over {(\sigma_{var} + y \cdot Poi )}} dy
```
Lets see what happends at big y
```math
\int {1 \over {\sqrt {2 \cdot \pi \cdot (|y| \cdot Poi )}}}  \cdot \exp {- |y|\over {Poi }} dy
```

It is a completely safe function, so we can incorporate Poisson noise, but we have to make it into a probability, i.e divide it by its integration. 

So, to do that I have to numerically integrate this function and we have to link to a library that can do just that.
